src/main/resources/db/migration/V5.13.0.3__8889-change-filepids-default.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO setting(content, lang, name) VALUES
+              ('true', '', ':FilePIDsEnabled')
+       ON CONFLICT DO NOTHING;
```

Comment:
I'm getting this error (I'm on 687c872b81):

```
dev_dataverse> Caused by: org.flywaydb.core.internal.sqlscript.FlywaySqlScriptException: Migration V5.13.0.3__8889-change-filepids-default.sql failed
dev_dataverse> ------------------------------------------------------------
dev_dataverse> SQL State  : 23514
dev_dataverse> Error Code : 0
dev_dataverse> Message    : ERROR: new row for relation "setting" violates check constraint "non_empty_lang"
dev_dataverse>   Detail: Failing row contains (1, true, , :FilePIDsEnabled).
dev_dataverse> Location   : db/migration/V5.13.0.3__8889-change-filepids-default.sql (/opt/payara/deployments/dataverse/WEB-INF/classes/db/migration/V5.13.0.3__8889-change-filepids-default.sql)
dev_dataverse> Line       : 1
dev_dataverse> Statement  : INSERT INTO setting(content, lang, name) VALUES
dev_dataverse>               ('true', '', ':FilePIDsEnabled')
dev_dataverse>        ON CONFLICT DO NOTHING
```

Reply:
This error goes away if you change empty string to NULL: https://github.com/IQSS/dataverse/pull/9721/files#r1270667037